GitTfs.Vs2015/paket.references

```diff
@@ -1,6 +1,2 @@
-//If you plan to update this file, you must first absolutly read our documentation on the subject
-//because there is some subtleties on how we use it in our project:
-//https://github.com/git-tfs/git-tfs/blob/master/doc/paket.md
```

Comment:
I also would like to keep comments.


Try update your local version of paket.exe because it's perhaps a recent development 

I will try myself tonight...